modules/label.rb

```diff
@@ -0,0 +1,17 @@
+require_relative '../item'
+
+class Label
+  attr_accessor :title
+
+  def initialize(title, color, id: nil)
+    @id = id.nil? ? Random.rand(1..1000) : id
```

Comment:
- [ ] Similar to `book.rb` I think it would be better if the `id` property of the label is generated automatically using `rand` and not passed as an argument in the `initialize` method. This will ensure less chances of labels having the same `id`